### ISSUE TITLE
Editor: Can't use keyboard's down arrow to scroll in a longer post (Simple Sites)

### ISSUE THREAD

**Issue by supernovia** [Automattic/wp-calypso#64752]
### Quick summary

The down arrow key does not work for scrolling in a longer post.

### Steps to reproduce

1. Start at wordpress.com/post on an account that has a simple site
2. Create a long post (I used Lorem)
3. Scroll to the top and bottom with your mouse
4. Try scrolling to the top, then to the bottom, using keyboard arrows

### What you expected to happen

I would expect the down arrow to scroll all the way down, every time.

### What actually happened

Sometimes down arrow only makes it part way through the post before it stops and won't scroll further. I caught it on video here:

https://user-images.githubusercontent.com/3476361/174411236-b3233498-4972-45f4-8193-6ab8c36d09a5.mov

### Context

Customer report at https://wordpress.com/forums/topic/why-cant-i-scroll-on-draft-posts-in-mac/

### Simple, Atomic or both?

Simple

### Theme-specific issue?

No

### Browser, operating system and other notes

Operating system: macOS Monterey version 12.4
Browser: Safari version 15.5 (1717613.2.7.1.8)

Also: macOS Big Sur 11.6.6
Safari 15.5
Chrome Version 102.0.5005.115

Oddly, after I restarted Chrome, it started scrolling properly in both Chrome and Safari. It seems after several attempts then just going to some other tab for a while, the scroll starts to work, but I'm not positive that's what's happening.

### Reproducibility

Intermittent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

Scroll using a mouse or trackpad instead.

**Comment by fstath** [comment]
The user the initially reported this came back with more information. Their post consists of only 239 words but it is split into 49 paragraphs.

Forum thread: https://wordpress.com/forums/topic/why-cant-i-scroll-on-draft-posts-in-mac/?view=all

**Comment by kerrynicl** [comment]
📌 HOUSEKEEPING
- Labels ✅ 
- Priority ✅ 
- Replicable on Core - No
- Updated main description of report - No

📌 SCRUBBING
- Tested on Simple ✅ 
- Tested on AT ✅ 
- Tested on Self-hosted ✅ 

📌 FINDINGS/SCREENSHOTS/VIDEO
I was able to replicate this on both Simple and Atomic sites. 

https://user-images.githubusercontent.com/83325600/175359811-cf69da43-c1ca-4d5f-beb8-50eb2ab41753.mp4

📌 ACTIONS
- Marked as Triaged for Quality Squad review
